<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphClass</class>
 <widget class="QMainWindow" name="GraphClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="imageLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionSauver"/>
    <addaction name="actionSauver_sous"/>
    <addaction name="actionFermer"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
    <addaction name="actionOuvrir_une_Image"/>
    <addaction name="separator"/>
    <addaction name="actionG_rer_les_villes"/>
    <addaction name="actionG_rer_les_routes"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionChoix_r_pertoire"/>
    <addaction name="actionRestaurer"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOutils"/>
   <addaction name="menuOption"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetGestionVilles">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="windowTitle">
    <string notr="true">Gestion des villes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonSaisiVilles">
       <property name="text">
        <string>Saisir une ville</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonSuppVille">
       <property name="text">
        <string>Supprimer une ville</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetGestionRoutes">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="windowTitle">
    <string notr="true">Gestion des routes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Saisir une route</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Supprimer une route</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetGestionParcours">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 255);</string>
   </property>
   <property name="windowTitle">
    <string notr="true">Gestion des parcours</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Effacer le parcours</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionSauver">
   <property name="text">
    <string>Sauver</string>
   </property>
  </action>
  <action name="actionSauver_sous">
   <property name="text">
    <string>Sauver sous</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionOuvrir_une_Image">
   <property name="text">
    <string>Ouvrir une Image</string>
   </property>
  </action>
  <action name="actionG_rer_les_villes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gérer les villes</string>
   </property>
  </action>
  <action name="actionG_rer_les_routes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gérer les routes</string>
   </property>
  </action>
  <action name="actionChoix_r_pertoire">
   <property name="text">
    <string>Choix répertoire</string>
   </property>
  </action>
  <action name="actionRestaurer">
   <property name="text">
    <string>Restaurer</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="graph.qrc"/>
 </resources>
 <connections/>
</ui>
